index.bs

```diff
@@ -4536,6 +4574,7 @@ failed</dfn> steps given |context| and |navigation status|:
         browsingContext.UserPromptClosedParameters = {
           context: browsingContext.BrowsingContext,
           accepted: bool,
+          type: browsingContext.UserPromptType,
```

Comment:
This field seems to be tricky to implement on our side. Can we move this part out of the scope of this PR and discuss it separately?

Reply:
UPD: as long as there can be not more than 1 open prompt, it's not an issue.